### ISSUE TITLE
Groups in frame stacking

### ISSUE THREAD

**Issue by seriouz** [brittyazel/EnhancedRaidFrames#50]
Can you add an option, so that the raid frame groups are stacked from bottom to top if they are horizontal?

**Comment by brittyazel** [comment]
I have no idea what you are asking. Are you talking about the indicator icons? Or do you want me to rearrange the raid frames themselves?

**Comment by Amissia** [comment]
I believe they're asking about the frames. Default Blizzard frames don't sort bottom -> top, they sort top -> bottom and there is no "grow up" or "grow down" options. The focus of this add-on is the indicators themselves, so this request doesn't really apply to the contents of the add-on. You may be able to find something else on Curse which does what you're asking to work alongside the indicators, but you'd have to look for it.

**Comment by brittyazel** [comment]
Yeah, I mean we do some very basic general stuff like scale, alpha, out of range distance, etc, but this seems pretty large of a change. I'm not sure it falls into the purview of this project

**Comment by seriouz** [comment]
The addon is called `EnhancedRaidFrames` and sorting of the default Blizzard frames from bottom to top or right to left are an enhancement over the the default options Blizzard has allowed/implemented in the options menu.

**Comment by brittyazel** [comment]
This isn't an easy request though. It's actually quite involved. I'll consider it but I'll have the weigh the amount of work it will take
